### ISSUE TITLE
Wishlist: add SVCB/HTTPS record types, and even used by default (at least for apex and `www` for HTTPS?)

### ISSUE THREAD

**Issue by pmevzek-godaddy** [dnsviz/dnsviz#156]
As more and more browsers use `HTTPS` records instead of `A`+`AAAA` ones to connect to websites, it would be great to have a default run with `HTTPS` records analysis on apex + `www`.

Or at the very least, to have both record types among the list of possible record types to query and let users select them if desired.

**Comment by pspacek** [comment]
I agree having HTTPS in the list would be very good. I'm just debugging a domain which misbehaves only on HTTPS qtype.
